src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt

```diff
@@ -0,0 +1,3 @@
+### sprite_set_velocity() usage example
+
+The following code is an example of using [sprite set velocity](https://splashkit.io/api/sprites/#sprite-set-velocity) which sets the velocity of a sprite on a splashkit window.
```

Comment:
Update this to be a bit clearer and use capital first letters on the function name.
eg. "The following code is an example of using [Sprite Set Velocity](https://splashkit.io/api/sprites/#sprite-set-velocity) which sets the velocity (speed and direction) of a sprite on a window.

Reply:
I have updated the description